engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/runtime/GuardedTruffleContext.java

```diff
@@ -27,6 +29,16 @@ public GuardedTruffleContext(TruffleContext context, boolean isSingleThreaded) {
     }
   }
 
+  /**
+   * Spawns new thread with associated {@code context}
+   *
+   * @param env environment to spawn the thread in
+   * @param run code to execute in given TruffleContext
+   */
+  public Thread createThread(TruffleLanguage.Env env, Consumer<TruffleContext> run) {
+    return env.createThread(() -> run.accept(context), context);
```

Comment:
I wonder if native-image will be complaining again about this one.

Reply:
Looks like all tests are OK.